src/codeview.js

```diff
@@ -585,7 +626,8 @@ var Codeview = GObject.registerClass({
         });
 
         if (this._numErrors !== results.length) {
```

Comment:
Instead of accessing `this._numErrors` directly, use `this.errors`. Here and everywhere `this._numErrors` appears.

Reply:
ok